src/init/consented/dfp-listeners.ts

```diff
@@ -7,14 +6,8 @@ const initDfpListeners = (): Promise<void> => {
 	window.googletag.cmd.push(() => {
 		const pubads = window.googletag.pubads();
 
-		pubads.addEventListener(
-			'slotRenderEnded',
-			raven.wrap<typeof onSlotRender>(onSlotRender),
-		);
-		pubads.addEventListener(
-			'slotOnload',
-			raven.wrap<typeof onSlotLoad>(onSlotLoad),
-		);
+		pubads.addEventListener('slotRenderEnded', onSlotRender);
+		pubads.addEventListener('slotOnload', onSlotLoad);
```

Comment:
I expect we'd still like these errors to be reportable to Sentry. What's the best way to achieve this in an idiomatic Sentry way?

Reply:
Sentry has [removed the `wrap` export](https://docs.sentry.io/platforms/javascript/migration/v7-to-v8/#removal-of-wrap-export), with no replacement API. Perhaps we could recreate something similar in `utils/report-error`

```js
// not tested!
export const wrapWithErrorReporting = (fn, tags) => {
  return function () {
    try {
      fn(arguments)
    } catch (e) {
      reportError(e, tags)
    }
}
```